tests/torchtune/modules/_export/test_attention.py

```diff
@@ -0,0 +1,230 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+import tempfile
+import unittest
+
+import torch
+from torch._inductor.package import load_package, package_aoti
+from torch.testing import assert_close
+from torchtune.models.llama3_1._position_embeddings import Llama3ScaledRoPE
+from torchtune.modules._export.attention import (
+    MultiHeadAttention as ExportMultiHeadAttention,
+)
+from torchtune.modules.attention import MultiHeadAttention as TTMultiHeadAttention
```

Comment:
This unit test should exist in a downstream library as per the original plan. Parity check tests (TTMultiHeadAttention == ExportMultiHeadAttention) can't exist in torchtune otherwise our PRs will be blocked whenever we make a change to MultiHeadAttention. Unit tests in torchtune (such as checking export) should be decoupled from implementations outside of the _export folder.